src/components/posts/buttons/EditModal.js

```diff
@@ -0,0 +1,44 @@
+import { useRef } from "react";
+import Post from "../../UI/Post";
+import classes from "./EditModal.module.css";
+
+function EditModal({ onClick, content, id, setRefreshPost }) {
```

Comment:
`onCancel` might be a better name here otherwise it isn't necessarily obvious whether this is the cancel or submit function

Reply:
yes